[rocm-dbgapi] Missing dependencies

Main errors:
```
Generating graphicash: dot: command not found
```
and
```
make[4]: pdflatex: No such file or directory
```
----
Full build log:
```
==> Starting build()...
-- The C compiler identification is GNU 10.2.0
-- The CXX compiler identification is GNU 10.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Found Threads: TRUE  
-- Looking for backtrace.h
-- Looking for backtrace.h - not found
-- Code Object Manager found at /opt/rocm/lib/cmake/amd_comgr.
-- Found Git: /usr/bin/git (found version "2.28.0") 
-- Found Doxygen: /usr/bin/doxygen (found version "1.8.20") found components: doxygen missing components: dot
-- Configuring done
-- Generating done
-- Build files have been written to: /build/rocm-dbgapi/src/build
make: Entering directory '/build/rocm-dbgapi/src/build'
make[1]: Entering directory '/build/rocm-dbgapi/src/build'
make[2]: Entering directory '/build/rocm-dbgapi/src/build'
make[3]: Entering directory '/build/rocm-dbgapi/src/build'
Scanning dependencies of target doc
make[3]: Leaving directory '/build/rocm-dbgapi/src/build'
make[3]: Entering directory '/build/rocm-dbgapi/src/build'
[100%] Generating documentation
warning: Tag 'TCL_SUBST' at line 257 of file '/build/rocm-dbgapi/src/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'CLANG_ASSISTED_PARSING' at line 1040 of file '/build/rocm-dbgapi/src/build/Doxyfile' belongs to an option that was not enabled at compile time.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u", or recompile doxygen with this feature enabled.
warning: Tag 'CLANG_OPTIONS' at line 1048 of file '/build/rocm-dbgapi/src/build/Doxyfile' belongs to an option that was not enabled at compile time.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u", or recompile doxygen with this feature enabled.
warning: Tag 'PERL_PATH' at line 2131 of file '/build/rocm-dbgapi/src/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'MSCGEN_PATH' at line 2153 of file '/build/rocm-dbgapi/src/build/Doxyfile' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
/build/rocm-dbgapi/src/build/include/amd-dbgapi.h:3660: warning: explicit link request to 'AMD_DBGAPI_ARCHITECTURE_INFO_WATCHPOINT_SHARE' could not be resolved
/build/rocm-dbgapi/src/build/include/amd-dbgapi.h:4553: warning: explicit link request to 'AMD_DBGAPI_ADDRESS_SPACE_LANE' could not be resolved
/build/rocm-dbgapi/src/build/include/amd-dbgapi.h:5045: warning: argument 'source_address_space' of command @param is not found in the argument list of amd_dbgapi_convert_address_space(amd_dbgapi_process_id_t process_id, amd_dbgapi_wave_id_t wave_id, amd_dbgapi_lane_id_t lane_id, amd_dbgapi_address_space_id_t source_address_space_id, amd_dbgapi_segment_address_t source_segment_address, amd_dbgapi_address_space_id_t destination_address_space_id, amd_dbgapi_segment_address_t *destination_segment_address)
/build/rocm-dbgapi/src/build/include/amd-dbgapi.h:5045: warning: argument 'destination_address_space' of command @param is not found in the argument list of amd_dbgapi_convert_address_space(amd_dbgapi_process_id_t process_id, amd_dbgapi_wave_id_t wave_id, amd_dbgapi_lane_id_t lane_id, amd_dbgapi_address_space_id_t source_address_space_id, amd_dbgapi_segment_address_t source_segment_address, amd_dbgapi_address_space_id_t destination_address_space_id, amd_dbgapi_segment_address_t *destination_segment_address)
/build/rocm-dbgapi/src/build/include/amd-dbgapi.h:5118: warning: The following parameters of amd_dbgapi_convert_address_space(amd_dbgapi_process_id_t process_id, amd_dbgapi_wave_id_t wave_id, amd_dbgapi_lane_id_t lane_id, amd_dbgapi_address_space_id_t source_address_space_id, amd_dbgapi_segment_address_t source_segment_address, amd_dbgapi_address_space_id_t destination_address_space_id, amd_dbgapi_segment_address_t *destination_segment_address) are not documented:
  parameter 'source_address_space_id'
  parameter 'destination_address_space_id'
Doxygen version used: 1.8.20
Notice: Output directory '/build/rocm-dbgapi/src/build/doc/' does not exist. I have created it for you.
Searching for include files...
Searching for files in directory /build/rocm-dbgapi/src/ROCdbgapi-rocm-3.7.0/include
Searching for example files...
Searching for images...
Searching for dot files...
Searching for msc files...
Searching for dia files...
Searching for files to exclude
Searching INPUT for files to process...
Reading and parsing tag files
Parsing files
Preprocessing /build/rocm-dbgapi/src/build/include/amd-dbgapi.h...
Parsing file /build/rocm-dbgapi/src/build/include/amd-dbgapi.h...
Building macro definition list...
Building group list...
Building directory list...
Building namespace list...
Building file list...
Building class list...
Computing nesting relations for classes...
Associating documentation with classes...
Building example list...
Searching for enumerations...
Searching for documented typedefs...
Searching for members imported via using declarations...
Searching for included using directives...
Searching for documented variables...
Building interface member list...
Building member list...
Searching for friends...
Searching for documented defines...
Computing class inheritance relations...
Computing class usage relations...
Flushing cached template relations that have become invalid...
Computing class relations...
Add enum values to enums...
Searching for member function documentation...
Creating members for template instances...
Building page list...
Search for main page...
Computing page relations...
Determining the scope of groups...
Sorting lists...
Determining which enums are documented
Computing member relations...
Building full member lists recursively...
Adding members to member groups.
Computing member references...
Inheriting documentation...
Generating disk names...
Adding source references...
Adding xrefitems...
Sorting member lists...
Setting anonymous enum type...
Computing dependencies between directories...
Generating citations page...
Counting members...
Counting data structures...
Resolving user defined references...
Finding anchors and sections in the documentation...
Transferring function references...
Combining using relations...
Adding members to index pages...
Correcting members for VHDL...
Generating style sheet...
Generating search indices...
Generating example documentation...
Generating file sources...
Generating code for file include/amd-dbgapi.h...
Generating file documentation...
Generating docs for file include/amd-dbgapi.h...
Generating page documentation...
Generating group documentation...
Generating class documentation...
Generating docs for compound amd_dbgapi_address_class_id_t...
Generating docs for compound amd_dbgapi_address_space_id_t...
Generating docs for compound amd_dbgapi_agent_id_t...
Generating docs for compound amd_dbgapi_architecture_id_t...
Generating docs for compound amd_dbgapi_breakpoint_id_t...
Generating docs for compound amd_dbgapi_callbacks_s...
Generating docs for compound amd_dbgapi_code_object_id_t...
Generating docs for compound amd_dbgapi_dispatch_id_t...
Generating docs for compound amd_dbgapi_displaced_stepping_id_t...
Generating docs for compound amd_dbgapi_event_id_t...
Generating docs for compound amd_dbgapi_process_id_t...
Generating docs for compound amd_dbgapi_queue_id_t...
Generating docs for compound amd_dbgapi_register_class_id_t...
Generating docs for compound amd_dbgapi_register_id_t...
Generating docs for compound amd_dbgapi_shared_library_id_t...
Generating docs for compound amd_dbgapi_watchpoint_id_t...
Generating docs for compound amd_dbgapi_watchpoint_list_t...
Generating docs for compound amd_dbgapi_wave_id_t...
Generating namespace index...
Generating graph info page...
Generating directory documentation...
Generating index page...
Generating page index...
Generating module index...
Generating namespace index...
Generating namespace member index...
Generating annotated compound index...
Generating alphabetical compound index...
Generating hierarchical class index...
Generating graphicash: dot: command not found
error: Problems running dot: exit code=127, command='dot', arguments='"/build/rocm-dbgapi/src/build/doc/html/amd-dbgapi_8h__incl.dot" -Tpng -o "/build/rocm-dbgapi/src/build/doc/html/amd-dbgapi_8h__incl.png"'
sh: dot: command not found
error: Problems running dot: exit code=127, command='dot', arguments='"/build/rocm-dbgapi/src/build/doc/html/structamd__dbgapi__watchpoint__list__t__coll__graph.dot" -Tpng -o "/build/rocm-dbgapi/src/build/doc/html/structamd__dbgapi__watchpoint__list__t__coll__graph.png"'
sh: dot: command not found
error: Problems running dot: exit code=127, command='dot', arguments='"/build/rocm-dbgapi/src/build/doc/rtf/structamd__dbgapi__watchpoint__list__t__coll__graph.dot" -Tpng -o "/build/rocm-dbgapi/src/build/doc/rtf/structamd__dbgapi__watchpoint__list__t__coll__graph.png"'
sh: dot: command not found
sh: dot: command not found
error: Problems running dot: exit code=127, command='dot', arguments='"/build/rocm-dbgapi/src/build/doc/rtf/amd-dbgapi_8h__incl.dot" -Tpng -o "/build/rocm-dbgapi/src/build/doc/rtf/amd-dbgapi_8h__incl.png"'
error: Problems running dot: exit code=127, command='dot', arguments='"/build/rocm-dbgapi/src/build/doc/html/graph_legend.dot" -Tpng -o "/build/rocm-dbgapi/src/build/doc/html/graph_legend.png"'
sh: dot: command not found
error: Problems running dot: exit code=127, command='dot', arguments='"/build/rocm-dbgapi/src/build/doc/latex/structamd__dbgapi__watchpoint__list__t__coll__graph.dot" -Tpdf -o "/build/rocm-dbgapi/src/build/doc/latex/structamd__dbgapi__watchpoint__list__t__coll__graph.pdf"'
sh: dot: command not found
error: Problems running dot: exit code=127, command='dot', arguments='"/build/rocm-dbgapi/src/build/doc/latex/amd-dbgapi_8h__incl.dot" -Tpdf -o "/build/rocm-dbgapi/src/build/doc/latex/amd-dbgapi_8h__incl.pdf"'
error: problems opening map file /build/rocm-dbgapi/src/build/doc/html/amd-dbgapi_8h__incl.map for inclusion in the docs!
If you installed Graphviz/dot after a previous failing run, 
try deleting the output directory and rerun doxygen.
error: problems opening map file /build/rocm-dbgapi/src/build/doc/html/structamd__dbgapi__watchpoint__list__t__coll__graph.map for inclusion in the docs!
If you installed Graphviz/dot after a previous failing run, 
try deleting the output directory and rerun doxygen.
error: problem writing FIG 0 figure!
l class hierarchy...
Generating member index...
Generating file index...
Generating file member index...
Generating example index...
finalizing index lists...
writing tag file...
Combining RTF output...
Running plantuml with JAVA...
Running dot...
Generating dot graphs using 11 parallel threads...
Running dot for graph 1/7
Running dot for graph 2/7
Running dot for graph 3/7
Running dot for graph 4/7
Running dot for graph 5/7
Running dot for graph 6/7
Running dot for graph 7/7
Patching output file 1/4
Patching output file 2/4
Patching output file 3/4
lookup cache used 413/65536 hits=2297 misses=414
finished...
make[4]: Entering directory '/build/rocm-dbgapi/src/build/doc/latex'
make[4]: pdflatex: No such file or directory
make[4]: *** [Makefile:8: refman.pdf] Error 127
make[4]: Leaving directory '/build/rocm-dbgapi/src/build/doc/latex'
make[3]: *** [CMakeFiles/doc.dir/build.make:84: doc/html/index.html] Error 2
make[3]: *** Deleting file 'doc/html/index.html'
make[3]: Leaving directory '/build/rocm-dbgapi/src/build'
make[2]: *** [CMakeFiles/Makefile2:96: CMakeFiles/doc.dir/all] Error 2
make[2]: Leaving directory '/build/rocm-dbgapi/src/build'
make[1]: *** [CMakeFiles/Makefile2:103: CMakeFiles/doc.dir/rule] Error 2
make[1]: Leaving directory '/build/rocm-dbgapi/src/build'
make: *** [Makefile:205: doc] Error 2
make: Leaving directory '/build/rocm-dbgapi/src/build'
==> ERROR: A failure occurred in build().
```

--- a/rocm-dbgapi/.SRCINFO
+++ b/rocm-dbgapi/.SRCINFO
@@ -1,14 +1,13 @@
 pkgbase = rocm-dbgapi
 	pkgdesc = Support library necessary for a debugger of AMD's GPUs
 	pkgver = 3.7.0
-	pkgrel = 1
+	pkgrel = 2
 	url = https://github.com/ROCm-Developer-Tools/ROCdbgapi
 	arch = x86_64
 	license = MIT
 	makedepends = cmake
 	makedepends = rocm-cmake
 	makedepends = git
-	makedepends = doxygen
 	depends = comgr
 	depends = hsa-rocr
 	source = rocm-dbgapi-3.7.0.tar.gz::https://github.com/ROCm-Developer-Tools/ROCdbgapi/archive/rocm-3.7.0.tar.gz

--- a/rocm-dbgapi/PKGBUILD
+++ b/rocm-dbgapi/PKGBUILD
@@ -1,13 +1,13 @@
 # Maintainer Torsten Keßler <t dot kessler at posteo dot de>
 pkgname=rocm-dbgapi
 pkgver=3.7.0
-pkgrel=1
+pkgrel=2
 pkgdesc="Support library necessary for a debugger of AMD's GPUs"
 arch=('x86_64')
 url='https://github.com/ROCm-Developer-Tools/ROCdbgapi'
 license=('MIT')
 depends=('comgr' 'hsa-rocr')
-makedepends=('cmake' 'rocm-cmake' 'git' 'doxygen')
+makedepends=('cmake' 'rocm-cmake' 'git')
 source=("$pkgname-$pkgver.tar.gz::$url/archive/rocm-$pkgver.tar.gz")
 sha256sums=('bdeaf81ea8a0ac861a697e435c72cbe767c291638be43f0d09116ad605dfee4f')
 _dirname=$(basename "$url")-$(basename "${source[0]}" ".tar.gz")
@@ -17,7 +17,7 @@ build() {
             -S "$_dirname" \
             -DCMAKE_INSTALL_PREFIX=/opt/rocm
 
-    make -C build doc all
+    make -C build
 }
 
 package() {

